### ISSUE TITLE
Add a cheat sheet

### ISSUE THREAD

**Issue by philippjfr** [holoviz/panel#2635]
Panel's API has grown a good bit and it would be nice to have an overview and quick reference page in the form of a cheat sheet.

**Comment by jbednar** [comment]
Definitely. I think the key for such a cheat sheet is what _not_ to put in it. I.e., I think it should be specifically formulated for a certain type of user and usage, and only mention the APIs and interfaces that we think are most relevant and useful for that type of user. My guess is that the most appropriate audience for the first cheat sheet should be a beginner, curating a small number of primitives and concepts that we think will help most people get started and get their first apps together while addressing some of their first stumbling blocks as the app gets more complex.

**Comment by Shahin-rmz** [comment]
except than cheat sheet, having some more examples are also useful for users like me who learn via examples.
